Include/internal/pycore_instruction_sequence.h

```diff
@@ -26,23 +29,29 @@ typedef struct {
     int i_offset;
 } _PyInstruction;
 
-typedef struct {
+typedef struct instruction_sequence {
+    PyObject_HEAD
     _PyInstruction *s_instrs;
     int s_allocated;
     int s_used;
 
     int s_next_free_label; /* next free label id */
+
     /* Map of a label id to instruction offset (index into s_instrs).
      * If s_labelmap is NULL, then each label id is the offset itself.
      */
     int *s_labelmap;       /* label id --> instr offset */
     int s_labelmap_size;
+
+    /* PyList of instruction sequences of nested functions */
+    PyObject *s_nested;
```

Comment:
Does this mean that we should implement gc `traverse` and `visit` methods?  I imagine it can't actually be cyclic, but I guess there's risk that it is in the presence of some bug, so maybe some `asserts` if not implementing gc?

Reply:
Yes - scroll down to inst_seq_traverse etc in the .c file.

Reply:
Oh, sorry I missed that ;)